rules/false_positives/kuma_cp.yara

```diff
@@ -0,0 +1,13 @@
+rule kuma_cp_binary : override {
+  meta:
+    description = "kuma-cp"
+    downgrade = "true"
+    malware_shellcode_hash = "high"
```

Comment:
I'd say medium or ignore

Reply:
Addressed in [`e716ba1` (#502)](https://github.com/chainguard-dev/malcontent/pull/502/commits/e716ba11a75e6cbd2e99b0f54f0d82f3deb46bef).